mobile/go.mod

```diff
@@ -204,6 +207,11 @@ replace github.com/lightninglabs/lightning-node-connect => ../
 // We want to format raw bytes as hex instead of base64. The forked version
 // allows us to specify that as an option. This is required for the indirect
 // taproot-assets dependency to function properly.
-replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-display v1.30.0-hex-display
+replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-display v1.34.2-hex-display
 
-go 1.22.3
+// Force faraday to be set to v0.2.13-alpha, as we'll else get an ambiguous import during `make mobile` of which
+// faraday/frdrpc to use. This forces the dependency to the faraday/frdrpc of the faraday v0.2.13-alpha release, where
+// as the litcli dependency uses the first actual separate tagged release for frdrpc v0.1.0
+replace github.com/lightninglabs/faraday => github.com/lightninglabs/faraday v0.2.13-alpha
```

Comment:
LEt me know if you have a better suggestion of how to handle this.

Reply:
I was able to build the `ios` and `android` libraries locally without any issues. Could the `go` version be the issue? I'm running `go version go1.23.4 darwin/arm64`.

I don't know enough about go dependencies to suggest any other solutions.

Reply:
Resolving this in favour for the new comment.